bin/main.rb

```diff
@@ -1,3 +1,72 @@
 # frozen_string_literal: true
 
-'Hello World!'
+require_relative('../lib/game_logic.rb')
+
+tic_tac = Game.new
+
+puts "\n\nGame Start!\n\n"
+
+puts 'Input player 1 name:'
+player1 = gets.chomp.capitalize
+p1 = tic_tac.create_player(player1)
+
+puts 'Input player 2 name:'
+player2 = gets.chomp.capitalize
+p2 = tic_tac.create_player(player2)
+
+puts "INSTRUCTIONS:\n\n"
+puts "Tic-tac-toe is a very fun game!\n#{player1} gets the 'X' and #{player2} the 'O'\n\n"
+puts "The first one to connect three X's O's sequence is the Winner!\n\n"
+puts "If you want to quit the game, type 'quit' after you choose names\n\n"
+puts 'Select a cell by typing the row (letter) followed by the column (number)'
+puts "For example: 'a1'\n\n"
+
+puts "Here's your Tic-tac-toe game board!\n\n"
+
+puts tic_tac.display_board
```

Comment:
**Optional** you can wrap some of the logic in this file into functions to make you code more readable.

Reply:
Done :-D